src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheck.java

```diff
@@ -73,18 +75,40 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * To configure the check to print warnings if an "@incomplete" tag is found,
- * and not print anything if it is not found:
+ * Example:
+ * </p>
+ * <pre>
+ * &#47;**
+ * * Valid
+ * * &#64;author Author
+ * * Invalid
+ * * &#64;author
+ * *&#47;
+ * public class Test{}
+ * </pre>
+ * <p>
+ * To configure the check to print warnings if an "@incomplete" tag is found and
+ * the check will report a violation, if an empty incomplete tag is found:
  * </p>
  * <pre>
  * &lt;module name="WriteTag"&gt;
  *   &lt;property name="tag" value="@incomplete"/&gt;
  *   &lt;property name="tagFormat" value="\S"/&gt;
- *   &lt;property name="severity" value="ignore"/&gt;
  *   &lt;property name="tagSeverity" value="warning"/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * &#47;**
+ * * Valid
```

Comment:
please refer to https://github.com/checkstyle/checkstyle/pull/8063#discussion_r408748813

Reply:
rewrote completely